### PR TITLE
Price per token should be ETH not the token name #1961

### LINKED ISSUE #1961: Price per token should be ETH not the token name
IMG_EB6B295654F4-1

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #1961

### CODE REVIEW

File: AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift [hunk 1]
```diff
@@ -49,7 +49,7 @@ class EnterSellTokensCardPriceQuantityViewController: UIViewController, TokenVer
         return viewModel.token.server
     }
     let assetDefinitionStore: AssetDefinitionStore
-    lazy var pricePerTokenField = AmountTextField(tokenObject: viewModel.token)
```
Comment (hboon) [inline AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift:52]: I think the fix might be simpler and more correct by replacing:

```
AmountTextField(tokenObject: viewModel.token)
```

with:

```
AmountTextField(tokenObject: ethToken)
```

And you can get ethToken via `TokensDataStore(forServer:)`.

Can you see if that makes sense and is right? The problem being in the send screen, we want the amount textfield for entering the amount of token the send screen is for. But in this screen, no matter which token it is, we want the amount textfield to be for entering Ether (or whichever native crypto token like xDai, etc).

**Comment by James-Sangalli** [comment]
@vladyslav-iosdev please use the main repo instead of a fork from now, cheers.

**Comment by hboon** [comment]
@vladyslav-iosdev is this good for review?

**Comment by vladyslav-iosdev** [comment]
> @vladyslav-iosdev is this good for review?

yes

**Pull request merged**